### PR TITLE
Prepare 0.3.0 release

### PR DESCRIPTION
**Pull request by thiemowmde**
I'm not sure if this is worth a major release. The breaking changes are in an abstract test class.

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
These abstract test classes are used by other components.

One can have breaking changes between 0.2 and 0.3, since it's in the 0.x range. We should however not still be in the 0.x range, so we could use this as excuse to do the bump.

**Comment by thiemowmde** [comment]
Ok, I will change to 1.0.0.

**Comment by thiemowmde** [comment]
Changed to 1.0.0.

File: composer.json [hunk 1]
```diff
@@ -25,11 +25,11 @@
 	"require": {
 		"php": ">=5.3.0",
 		"data-values/data-values": "~1.0|~0.1",
-		"data-values/interfaces": "~0.1.4"
+		"data-values/interfaces": "~1.0"
```
Comment (JeroenDeDauw) [inline composer.json]: Since that is not released yet, use `~1.0@dev`

**Comment by JeroenDeDauw** [comment]
Generally looks good

**Comment by thiemowmde** [comment]
Going back to 0.3.0 for the reason explained in https://github.com/DataValues/Interfaces/pull/14.

**Pull request merged**